mocha.js

```diff
@@ -1464,6 +1464,7 @@ Mocha.prototype.ui = function (name) {
     exports.before = context.before || context.suiteSetup;
     exports.describe = context.describe || context.suite;
     exports.it = context.it || context.test;
+    exports.xit = context.xit || context.test;
```

Comment:
Small tip: you don't actually have to update the `mocha.js` file at the base of the project, as it is generated from the other files as part of the publication process. (On the other hand, for the same reason, you don't have to not update it either -- it will simply be overwritten later, as long as you update the other files in addition. I just mention this because it's useful to know you don't have search through the gigantic bundle for the code to update and do have to update other files for changes to actually be picked up.)

Reply:
@ScottFreeCode Updated the PR with suggested changes. This time not updated mocha.js at root level.